### ISSUE TITLE
Resources not being given up, weird behavior

### ISSUE THREAD

**Issue by alidan** [deanoemcke/thegreatsuspender#424]
Ok, I installed a tab called sidewise, this screwed everything up I think as tabs use to not bunch together, now they do. 

When I suspend a tab, after a while they start to refuse to give up resources, to make it simple, I have about 70 tab groups that have one active tab, followed by what should be 1-8 suspended tabs, but the suspended tabs don't consolidate into the extension.

Any help with fixing this would be greatly appreciated.
I honestly doubt that this is a specific extension issue, but some kind of compatibility with sidewise (which I removed but it still does this crap)
Hell, if tabs just stopped grouping together this issue would likely be solved.

**Comment by deanoemcke** [comment]
I'm finding it hard to understand exactly what you are trying to explain sorry. Perhaps a screenshot would help?
Is this still an issue? What version of the extension are you using?

**Comment by alidan** [comment]
sorry bout that, just did an email search and I never received a notification for this till it was closed.

The issue was simple, I started to use this extention
https://chrome.google.com/webstore/detail/sidewise-tree-style-tabs/biiammgklaefagjclmnlialkmaemifgo?hl=en
shortly after that, in the chrome task manager I would have 1 tab that was suspended, and 1-20 tabs that were in the same process, and even when I suspended the active tab that everything was under, none of the tabs would give up any of their ram resources.

I believe it was an incompatibility with that extension that caused it. since then I moved from sidewise to tabs outliner and haven't seen the issue replicate itself. 

since the first post of this issue, I have moved to a new computer so the ram issue is far more minimal then it was before, having 32gb of ram rather than 16gb, with somewhere around 550 tabs suspended, the extension wants 1gb in its footprint and 900mb active use.

as for version I was using, whatever is on the chrome web store at the time, and still now.

Either way, its no longer an issue for me as I moved away from the extension I believe was causing problems.
